package-lock.json

```diff
@@ -11,7 +11,7 @@
 				"@aws-amplify/ui-react": "3.6.1",
 				"@aws-sdk/credential-providers": "3.370.0",
 				"@turf/turf": "6.5.0",
-				"@types/react-window": "^1.8.5",
+				"@types/react-window": "1.8.5",
```

Comment:
are the modifications on package.json fine or should we not modify this file unless we want to bring in a new dependency?

Reply:
This is `package-lock.json`, not `package.json`